app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/assets/PoolUnitsTab.kt

```diff
@@ -153,6 +167,4 @@ fun PoolResourcesValues(poolUnit: PoolUnit, modifier: Modifier = Modifier) {
 }
 
 @Composable
-fun PoolUnit.name() = displayTitle.ifEmpty {
-    stringResource(id = R.string.account_poolUnits_unknownPoolUnitName)
-}
+fun PoolUnit.name() = displayTitle
```

Comment:
Maybe no need for this method ? Up to you

Reply:
Wanted to explicitly create a method for that so in the future if someone thinks that we forgot to catch the empty pool's name it was done on purpose.